src/Helpers.purs

```diff
@@ -130,3 +135,20 @@ appendRightHashMap
   -> HashMap k v
   -> HashMap k v
 appendRightHashMap = HashMap.unionWith (flip const)
+
+-- | Ensures that given predicate does not hold for a value.
+never
```

Comment:
Personally, I'm not sure if the name of this function communicates its meaning very clearly

Reply:
It feels nice to me. I've seen something similar on pursuit as well. Any propositions?